<launch>
  <node pkg="path_sim" type="obstacle_detection" name="obstacle_detection" output="screen" respawn="false">
  <param name="odom_topic" value="/mobile_base/commands/velocity/t" />
  <param name="robot_width" value="0.5" />
  <param name="detection_length" value="2.0" />
  <param name="road_min" value="5.0" />
  <param name="frequency" value="5" />
  <param name="localmap_topic" value="/move_base/global_costmap/costmap" />
  <param name="odom_topic" value="/odom_localization" />
  </node>
</launch>
